### PR TITLE
fix: add rehype-pretty-code dark mode support #930

### LINKED ISSUE #930: feat: add `rehype-pretty-code` dark mode support
### Description

Right now, the `pre` code block doesn't support dark mode. It would be great to have a dark theme.

Documentation: https://rehype-pretty.pages.dev/#multiple-themes-dark-and-light-mode

Open for theme suggestions.

CleanShot 2024-07-12 at 22 20 06@2x
CleanShot 2024-07-12 at 22 19 44@2x

### PR DESCRIPTION
**Pull request by Ratnadeep2k**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation

### Before this PR

{Please add a screenshot here}

### After this PR

{Please add a screenshot here}

### CODE REVIEW

File: apps/web/src/components/content/pre.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import { Button } from "@openstatus/ui";
 import { Clipboard, ClipboardCopy } from "lucide-react";
 import React from "react";
-
+import rehypePrettyCode from "rehype-pretty-code";
```
Comment (mxkaske) [inline apps/web/src/components/content/pre.tsx]: please remove

**Review by mxkaske — COMMENTED** [review]
Please remove the changes on `packages.json`, `pnpm-lock.yaml` and the single import. Otherwise lgtm!

**Comment by Ratnadeep2k** [comment]
@mxkaske Please review

**Comment by mxkaske** [comment]
Thanks @Ratnadeep2k!

**Pull request merged**